pkg/core/bootstrap/bootstrap.go

```diff
@@ -169,7 +169,7 @@ func buildRuntime(appCtx context.Context, cfg kuma_cp.Config) (core_runtime.Runt
 		return nil, err
 	}
 
-	initializeTokenIssuers(builder)
+	initializeTokenIssuers(appCtx, builder)
```

Comment:
To me this doesn't seem needed

Reply:
Isn't it needed for signing key managers which in turn are needed when a new tenant joins and we need to create ingress token signing key and zone token signing key?